Functions/QualityControl/xASL_qc_CollectParameters.m

```diff
@@ -98,8 +102,10 @@
         x.Output.(ScanType).Version_ExploreASL = x.Output.SoftwareVersion.ExploreASL;
         x.Output.(ScanType).Version_Matlab = x.Output.SoftwareVersion.Matlab;
         x.Output.(ScanType).Version_SPM12 = x.Output.SoftwareVersion.SPM12;        
-    case {'ASL', 'dwi', 'func'}
+    case {'dwi', 'func'}
```

Comment:
Also with respect to the issue at the end:
OK. But dwi, func, structural will still have the SESSIONS and iSESSION parameters assigned. So while only a single session is allowed, I can already adapt the QC so that it would work with more sessions. Do you agree? If yes, I'll implement this now - rather than postponing it and making the code inconsistent.

Reply:
Yes, I agree. This makes sense

Reply:
Done in 0b7a798c

Reply:
No, they should all have sessions except for structural here.

Reply:
Yes. That is what I did. ASL and func have now sessions. Dwi doesn't have "sessions" in QC as dwi doesn't have a dedicated QC :)

Reply:
It definitively does. In the same way as ASL. I've just coded that. Or I miss something?

Reply:
Haha in this very line where we are discussing it (line 103), both dwi & func get 

x.Output.(ScanType).Version_FSL

instead of 

x.Output.ASL.(x.SESSIONS{iSession}).Version_FSL

Reply:
Done in 43a10227